carWarGame.py

```diff
@@ -18,13 +20,13 @@ class Deck:
     suits = ['♣','♦','♥','♠']
 
     def __init__(self):
-       self.cards = self.generateDeck(Deck.ranks,Deck.suits)
+       self.cards = self.generateDeck()
 
 
-    def generateDeck(self,ranks, suits):
+    def generateDeck(self) -> list:
```

Comment:
```suggestion
    def generate_deck(self) -> list:
```

Python functions and variables are defined with **snake_case**

Also, please include a docstring for your functions.

Reply:
Yes sir, I got confused with JS. Won't happen again.